Error when creating mini RADI

A error appears in the 23 step. Here are the logs:
```bash
 => ERROR [23/34] RUN git clone -b v2.4.1 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git /tmp/MicroXRCEAgent   && cd /tmp/MicroXRCEAgent   && mkdir build && cd build     23.7s
------
 > [23/34] RUN git clone -b v2.4.1 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git /tmp/MicroXRCEAgent   && cd /tmp/MicroXRCEAgent   && mkdir build && cd build   && cmake ..   && make -j $(nproc)   && make install   && ldconfig /usr/local/lib/   && rm -rf /tmp/*:
0.285 Cloning into '/tmp/MicroXRCEAgent'...
2.370 Note: switching to '40954c2379dfcb2bfa9f6ba22146c57c6742b5b7'.
2.370 
2.370 You are in 'detached HEAD' state. You can look around, make experimental
2.370 changes and commit them, and you can discard any commits you make in this
2.370 state without impacting any branches by switching back to a branch.
2.370 
2.370 If you want to create a new branch to retain commits you create, you may
2.370 do so (now or later) by using -c with the switch command. Example:
2.370 
2.370   git switch -c <new-branch-name>
2.370 
2.370 Or undo this operation with:
2.370 
2.370   git switch -
2.370 
2.370 Turn off this advice by setting config variable advice.detachedHead to false
2.370 
2.436 -- The C compiler identification is GNU 11.4.0
2.440 -- Detecting C compiler ABI info
2.496 -- Detecting C compiler ABI info - done
2.500 -- Check for working C compiler: /usr/bin/cc - skipped
2.500 -- Detecting C compile features
2.500 -- Detecting C compile features - done
2.540 -- The CXX compiler identification is GNU 11.4.0
2.544 -- Detecting CXX compiler ABI info
2.607 -- Detecting CXX compiler ABI info - done
2.611 -- Check for working CXX compiler: /usr/bin/c++ - skipped
2.611 -- Detecting CXX compile features
2.611 -- Detecting CXX compile features - done
2.658 -- Looking for pthread.h
2.709 -- Looking for pthread.h - found
2.709 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
2.760 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
2.761 -- Found Threads: TRUE  
2.766 -- Configuring done
2.770 -- Generating done
2.770 -- Build files have been written to: /tmp/MicroXRCEAgent/build
2.803 [  2%] Creating directories for 'microxrcedds_client'
2.804 [  5%] Creating directories for 'fastcdr'
2.864 [  8%] Performing download step (git clone) for 'fastcdr'
2.864 [ 11%] Performing download step (git clone) for 'microxrcedds_client'
2.876 Cloning into 'microxrcedds_client'...
2.876 Cloning into 'fastcdr'...
5.161 HEAD is now at 2e55b30 Fix doxygen (#151)
5.202 [ 14%] Performing update step for 'fastcdr'
5.229 [ 17%] No patch step for 'fastcdr'
5.253 [ 20%] Performing configure step for 'fastcdr'
5.257 loading initial cache file /tmp/MicroXRCEAgent/build/fastcdr/tmp/fastcdr-cache-.cmake
5.257 -- Setting build type to 'Release' as none was specified.
5.285 -- The CXX compiler identification is GNU 11.4.0
5.289 -- Detecting CXX compiler ABI info
5.329 -- Detecting CXX compiler ABI info - done
5.333 -- Check for working CXX compiler: /usr/bin/c++ - skipped
5.333 -- Detecting CXX compile features
5.333 -- Detecting CXX compile features - done
5.333 -- Configuring Fast CDR
5.333 -- Version: 1.1.0
5.333 -- To change the version modify the file configure.ac
5.334 -- Performing Test SUPPORTS_NOTFOUND
5.372 -- Performing Test SUPPORTS_NOTFOUND - Success
5.372 -- Enforced cxx_std_11 CMake feature
5.373 -- Looking for C++ include sys/types.h
5.430 -- Looking for C++ include sys/types.h - found
5.430 -- Looking for C++ include stdint.h
5.493 -- Looking for C++ include stdint.h - found
5.493 -- Looking for C++ include stddef.h
5.552 -- Looking for C++ include stddef.h - found
5.552 -- Check size of long double
5.604 -- Check size of long double - done
5.604 -- Check size of __float128
5.659 -- Check size of __float128 - done
5.690 -- The C compiler identification is GNU 11.4.0
5.694 -- Detecting C compiler ABI info
5.759 -- Detecting C compiler ABI info - done
5.766 -- Check for working C compiler: /usr/bin/cc - skipped
5.766 -- Detecting C compile features
5.767 -- Detecting C compile features - done
5.774 -- Configuring done
5.777 -- Generating done
5.777 -- Build files have been written to: /tmp/MicroXRCEAgent/build/fastcdr/src/fastcdr-build
5.789 [ 23%] Performing build step for 'fastcdr'
5.829 [ 14%] Building CXX object src/cpp/CMakeFiles/fastcdr.dir/Cdr.cpp.o
5.829 [ 28%] Building CXX object src/cpp/CMakeFiles/fastcdr.dir/FastCdr.cpp.o
5.829 [ 42%] Building CXX object src/cpp/CMakeFiles/fastcdr.dir/FastBuffer.cpp.o
5.830 [ 71%] Building CXX object src/cpp/CMakeFiles/fastcdr.dir/exceptions/Exception.cpp.o
5.830 [ 71%] Building CXX object src/cpp/CMakeFiles/fastcdr.dir/exceptions/NotEnoughMemoryException.cpp.o
5.830 [ 85%] Building CXX object src/cpp/CMakeFiles/fastcdr.dir/exceptions/BadParamException.cpp.o
6.039 HEAD is now at 9717530 Release v2.4.1
6.068 [ 26%] Performing update step for 'microxrcedds_client'
6.080 [ 29%] No patch step for 'microxrcedds_client'
6.091 [ 32%] Performing configure step for 'microxrcedds_client'
6.098 loading initial cache file /tmp/MicroXRCEAgent/build/microxrcedds_client/tmp/microxrcedds_client-cache-.cmake
6.124 -- The C compiler identification is GNU 11.4.0
6.129 -- Detecting C compiler ABI info
6.162 -- Detecting C compiler ABI info - done
6.166 -- Check for working C compiler: /usr/bin/cc - skipped
6.166 -- Detecting C compile features
6.166 -- Detecting C compile features - done
6.192 -- The CXX compiler identification is GNU 11.4.0
6.196 -- Detecting CXX compiler ABI info
6.234 -- Detecting CXX compiler ABI info - done
6.238 -- Check for working CXX compiler: /usr/bin/c++ - skipped
6.238 -- Detecting CXX compile features
6.238 -- Detecting CXX compile features - done
6.257 -- Configuring done
6.259 -- Generating done
6.259 -- Build files have been written to: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build
6.269 [ 35%] Performing build step for 'microxrcedds_client'
6.285 [100%] Linking CXX shared library libfastcdr.so
6.289 [  6%] Creating directories for 'microcdr'
6.325 [ 12%] Performing download step (git clone) for 'microcdr'
6.331 [100%] Built target fastcdr
6.334 Cloning into 'microcdr'...
6.348 [ 38%] Performing install step for 'fastcdr'
6.367 Consolidate compiler generated dependencies of target fastcdr
6.378 [100%] Built target fastcdr
6.397 Install the project...
6.406 -- Install configuration: "Release"
6.406 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr
6.406 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/FastBuffer.h
6.406 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/eProsima_auto_link.h
6.407 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/Cdr.h
6.407 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/FastCdr.h
6.407 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/fastcdr_dll.h
6.407 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/exceptions
6.407 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/exceptions/Exception.h
6.408 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/exceptions/NotEnoughMemoryException.h
6.408 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/exceptions/BadParamException.h
6.408 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/include/fastcdr/config.h
6.408 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/lib/libfastcdr.so.1.1.0
6.409 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/lib/libfastcdr.so
6.409 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/lib/cmake/fastcdr/fastcdr-shared-targets.cmake
6.409 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/lib/cmake/fastcdr/fastcdr-shared-targets-release.cmake
6.409 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/lib/cmake/fastcdr/fastcdr-config.cmake
6.409 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/lib/cmake/fastcdr/fastcdr-config-version.cmake
6.409 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/fastcdr-1.1.0/share/fastcdr/LICENSE
6.428 [ 41%] Performing test step for 'fastcdr'
6.454 [ 44%] Completed 'fastcdr'
6.471 [ 44%] Built target fastcdr
6.480 [ 47%] Creating directories for 'fastdds'
6.539 [ 50%] Performing download step (git clone) for 'fastdds'
6.557 Cloning into 'fastdds'...
7.373 HEAD is now at 3d1b177 Release v2.0.1
7.437 [ 18%] Performing update step for 'microcdr'
7.461 [ 25%] No patch step for 'microcdr'
7.483 [ 31%] Performing configure step for 'microcdr'
7.492 loading initial cache file /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/microcdr/tmp/microcdr-cache-.cmake
7.492 -- Setting build type to 'Release' as none was specified.
7.538 -- The C compiler identification is GNU 11.4.0
7.545 -- Detecting C compiler ABI info
7.611 -- Detecting C compiler ABI info - done
7.619 -- Check for working C compiler: /usr/bin/cc - skipped
7.619 -- Detecting C compile features
7.620 -- Detecting C compile features - done
7.660 -- The CXX compiler identification is GNU 11.4.0
7.663 -- Detecting CXX compiler ABI info
7.723 -- Detecting CXX compiler ABI info - done
7.727 -- Check for working CXX compiler: /usr/bin/c++ - skipped
7.727 -- Detecting CXX compile features
7.727 -- Detecting CXX compile features - done
7.732 -- Configuring done
7.734 -- Generating done
7.734 -- Build files have been written to: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/microcdr/src/microcdr-build
7.748 [ 37%] Performing build step for 'microcdr'
7.783 [ 12%] Creating directories for 'ucdr'
7.853 [ 25%] No download step for 'ucdr'
7.877 [ 37%] No update step for 'ucdr'
7.904 [ 50%] No patch step for 'ucdr'
7.923 [ 62%] Performing configure step for 'ucdr'
7.930 loading initial cache file /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/microcdr/src/microcdr-build/ucdr-prefix/tmp/ucdr-cache-Release.cmake
7.934 -- Configuring done
7.936 -- Generating done
7.936 -- Build files have been written to: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/microcdr/src/microcdr-build
7.944 [ 75%] Performing build step for 'ucdr'
7.979 [ 16%] Building C object CMakeFiles/microcdr.dir/src/c/common.c.o
7.979 [ 33%] Building C object CMakeFiles/microcdr.dir/src/c/types/basic.c.o
7.979 [ 50%] Building C object CMakeFiles/microcdr.dir/src/c/types/string.c.o
7.979 [ 66%] Building C object CMakeFiles/microcdr.dir/src/c/types/array.c.o
7.980 [ 83%] Building C object CMakeFiles/microcdr.dir/src/c/types/sequence.c.o
8.227 [100%] Linking C shared library libmicrocdr.so
8.266 [100%] Built target microcdr
8.288 No install step for 'ucdr'
8.309 Completed 'ucdr'
8.338 Built target ucdr
8.366 [ 43%] Performing install step for 'microcdr'
8.393 Consolidate compiler generated dependencies of target microcdr
8.402 [100%] Built target microcdr
8.420 Install the project...
8.429 -- Install configuration: "Release"
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/lib/libmicrocdr.so.2.0.1
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/lib/libmicrocdr.so.2.0
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/lib/libmicrocdr.so
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/include/ucdr
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/include/ucdr/visibility.h
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/include/ucdr/microcdr.h
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/include/ucdr/config.h
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/share/microcdr/cmake/microcdrTargets.cmake
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/share/microcdr/cmake/microcdrTargets-release.cmake
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/share/microcdr/cmake/microcdrConfig.cmake
8.429 -- Installing: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/temp_install/microcdr-2.0.1/share/microcdr/cmake/microcdrConfigVersion.cmake
8.446 [ 50%] Completed 'microcdr'
8.478 [ 50%] Built target microcdr
8.498 [ 56%] Creating directories for 'uclient'
8.570 [ 62%] No download step for 'uclient'
8.596 [ 68%] No update step for 'uclient'
8.629 [ 75%] No patch step for 'uclient'
8.654 [ 81%] Performing configure step for 'uclient'
8.663 loading initial cache file /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build/uclient-prefix/tmp/uclient-cache-.cmake
8.667 -- Setting build type to 'Release' as none was specified.
8.671 -- Looking for pthread.h
8.735 -- Looking for pthread.h - found
8.735 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
8.798 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
8.799 -- Found Threads: TRUE  
8.800 -- Configuring done
8.804 -- Generating done
8.804 -- Build files have been written to: /tmp/MicroXRCEAgent/build/microxrcedds_client/src/microxrcedds_client-build
8.821 [ 87%] Performing build step for 'uclient'
8.862 [  2%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/input_best_effort_stream.c.o
8.863 [  5%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/input_reliable_stream.c.o
8.863 [  8%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/output_best_effort_stream.c.o
8.863 [ 11%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/stream_storage.c.o
8.863 [ 17%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/stream_id.c.o
8.864 [ 17%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/output_reliable_stream.c.o
8.864 [ 20%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/stream/seq_num.c.o
8.864 [ 23%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/session.c.o
8.866 [ 26%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/session_info.c.o
8.867 [ 32%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/submessage.c.o
8.867 [ 32%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/object_id.c.o
8.868 [ 35%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/util/ping.c.o
8.868 [ 41%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/serialization/xrce_header.c.o
8.868 [ 41%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/serialization/xrce_types.c.o
8.868 [ 44%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/create_entities_xml.c.o
8.869 [ 47%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/serialization/xrce_subheader.c.o
8.869 [ 50%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/create_entities_bin.c.o
8.869 [ 52%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/util/time.c.o
8.869 [ 55%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/common_create_entities.c.o
8.870 [ 61%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/write_access.c.o
8.870 [ 61%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/create_entities_ref.c.o
8.870 [ 64%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/core/session/read_access.c.o
8.871 [ 67%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/ip/tcp/tcp_transport.c.o
8.871 [ 70%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/ip/udp/udp_transport.c.o
8.872 [ 73%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/stream_framing/stream_framing_protocol.c.o
8.872 [ 76%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/ip/udp/udp_transport_posix.c.o
8.872 [ 79%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/serial/serial_transport_posix.c.o
8.873 [ 85%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/serial/serial_transport.c.o
8.873 [ 85%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/discovery/discovery.c.o
8.873 [ 91%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/ip/ip_posix.c.o
8.873 [ 91%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/ip/tcp/tcp_transport_posix.c.o
8.894 [ 94%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/discovery/transport/udp_transport_datagram_posix.c.o
8.895 [ 97%] Building C object CMakeFiles/microxrcedds_client.dir/src/c/profile/transport/custom/custom_transport.c.o
9.223 [100%] Linking C shared library libmicroxrcedds_client.so
9.257 [100%] Built target microxrcedds_client
9.271 No install step for 'uclient'
9.293 Completed 'uclient'
9.324 Built target uclient
9.355 [ 52%] Performing install step for 'microxrcedds_client'
9.385 Consolidate compiler generated dependencies of target microxrcedds_client
9.397 [100%] Built target microxrcedds_client
9.419 Install the project...
9.428 -- Install configuration: "Release"
9.428 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/lib/libmicroxrcedds_client.so.2.4.1
9.429 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/lib/libmicroxrcedds_client.so.2.4
9.429 -- Set runtime path of "/tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/lib/libmicroxrcedds_client.so.2.4.1" to ""
9.429 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/lib/libmicroxrcedds_client.so
9.429 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/visibility.h
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/multithread
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/multithread/multithread.h
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/discovery
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/discovery/discovery.h
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/stream_framing
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/stream_framing/stream_framing_protocol.h
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/custom
9.430 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/custom/custom_transport.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/ip.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp/udp_transport_posix_nopoll.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp/udp_transport_posix.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp/udp_transport_windows.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp/udp_transport_freertos_plus_tcp.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp/udp_transport.h
9.431 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/udp/udp_transport_rtems_bsd_net.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp/tcp_transport_windows.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp/tcp_transport_posix.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp/tcp_transport_posix_nopoll.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp/tcp_transport_rtems_bsd_net.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp/tcp_transport.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/ip/tcp/tcp_transport_freertos_plus_tcp.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/serial
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/serial/serial_transport_posix.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/serial/serial_transport_rtems_bsd_net.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/serial/serial_transport.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/serial/serial_transport_platform.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/can
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/can/can_transport.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/profile/transport/can/can_transport_posix.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/defines.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/client.h
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session
9.432 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/create_entities_bin.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/object_id.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/time_sync.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/session_info.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/session.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/output_best_effort_stream.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/input_reliable_stream.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/reliable_stream.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/seq_num.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/stream_storage.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/output_reliable_stream.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/input_best_effort_stream.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/stream/stream_id.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/create_entities_xml.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/create_entities_ref.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/common_create_entities.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/read_access.h
9.433 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/session/write_access.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/type
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/type/xrce_types.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/communication
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/core/communication/communication.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/transport.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/util
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/util/time.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/util/ping.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/include/uxr/client/config.h
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/share/microxrcedds_client/cmake/microxrcedds_clientTargets.cmake
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/share/microxrcedds_client/cmake/microxrcedds_clientTargets-release.cmake
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/share/microxrcedds_client/cmake/microxrcedds_clientConfig.cmake
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/share/microxrcedds_client/cmake/microxrcedds_clientConfigVersion.cmake
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/lib
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/lib/libmicrocdr.so.2.0.1
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/lib/libmicrocdr.so.2.0
9.434 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/lib/libmicrocdr.so
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/include
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/include/ucdr
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/include/ucdr/visibility.h
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/include/ucdr/config.h
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/include/ucdr/microcdr.h
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share/microcdr
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share/microcdr/cmake
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share/microcdr/cmake/microcdrConfig.cmake
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share/microcdr/cmake/microcdrConfigVersion.cmake
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share/microcdr/cmake/microcdrTargets.cmake
9.435 -- Installing: /tmp/MicroXRCEAgent/build/temp_install/microxrcedds_client-2.4.1/..//microcdr-2.0.1/share/microcdr/cmake/microcdrTargets-release.cmake
9.447 [ 55%] Completed 'microxrcedds_client'
9.485 [ 55%] Built target microxrcedds_client
23.68 fatal: invalid reference: 2.11.x
23.68 CMake Error at /tmp/MicroXRCEAgent/build/fastdds/tmp/fastdds-gitclone.cmake:40 (message):
23.68   Failed to checkout tag: '2.11.x'
23.68 
23.68 
23.69 make[2]: *** [CMakeFiles/fastdds.dir/build.make:101: fastdds/src/fastdds-stamp/fastdds-download] Error 1
23.69 make[1]: *** [CMakeFiles/Makefile2:141: CMakeFiles/fastdds.dir/all] Error 2
23.69 make: *** [Makefile:91: all] Error 2
------

 5 warnings found (use --debug to expand):
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 5)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 18)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 19)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 20)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 132)
Dockerfile.dependencies_humble:176
--------------------
 175 |     
 176 | >>> RUN git clone -b v2.4.1 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git /tmp/MicroXRCEAgent \
 177 | >>>   && cd /tmp/MicroXRCEAgent \
 178 | >>>   && mkdir build && cd build \
 179 | >>>   && cmake .. \
 180 | >>>   && make -j $(nproc) \
 181 | >>>   && make install \
 182 | >>>   && ldconfig /usr/local/lib/ \
 183 | >>>   && rm -rf /tmp/*
 184 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c git clone -b v2.4.1 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git /tmp/MicroXRCEAgent   && cd /tmp/MicroXRCEAgent   && mkdir build && cd build   && cmake ..   && make -j $(nproc)   && make install   && ldconfig /usr/local/lib/   && rm -rf /tmp/*" did not complete successfully: exit code: 2
Docker Base Image Build FAILED...exiting
```

--- a/scripts/mini_RADI/Dockerfile.dependencies_humble
+++ b/scripts/mini_RADI/Dockerfile.dependencies_humble
@@ -173,7 +173,7 @@ RUN git clone -b v1.14.0 https://github.com/PX4/PX4-Autopilot.git --recursive \
 # https://github.com/JdeRobot/RoboticsAcademy/issues/2310
 RUN echo 'param set-default COM_OF_LOSS_T 5' >> /PX4-Autopilot/build/px4_sitl_default/etc/init.d-posix/px4-rc.params
 
-RUN git clone -b v2.4.1 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git /tmp/MicroXRCEAgent \
+RUN git clone -b v2.4.3 https://github.com/eProsima/Micro-XRCE-DDS-Agent.git /tmp/MicroXRCEAgent \
   && cd /tmp/MicroXRCEAgent \
   && mkdir build && cd build \
   && cmake .. \

